examples/demo/planet.rs

```diff
@@ -16,6 +16,7 @@ impl Planet {
     pub fn new() -> Self {
         Self {
             noise: noise::Fbm::new()
+                .set_seed(1337 * 1337)
```

Comment:
Was this included intentionally?

Reply:
As the `noise` crate dependency changed, it produced different noise, so I tried different seeds until there was terrain under the camera.